README.md

```diff
@@ -17,7 +17,7 @@ To build the sample, run the following commands in the project root folder
 
 ```
 npm install
-node_modules/.bin/webpack --config webpack.config.js --dojoRoot <dojoRoot>
+node_modules/.bin/webpack --config webpack.config.js --env.dojoRoot <dojoRoot>
```

Comment:
Well, dojoRoot needs to be specified as a command line arg.  Not sure how to pass that as a parameter to the script